install-debuntu.sh

```diff
@@ -4,8 +4,8 @@ shopt -s extglob
 
 # Lists of supported architectures, Debian, and Ubuntu releases
 SUPPORTED_ARCHITECTURES='@(amd64|armhf|arm64)'
-SUPPORTED_DEBIAN_RELEASES='@(bullseye|bookworm)'
-SUPPORTED_UBUNTU_RELEASES='@(focal|jammy|noble)'
+SUPPORTED_DEBIAN_RELEASES='@(buster|bullseye|bookworm)'
```

Comment:
We no longer support the releases you are adding here. Please do not adjust these.